index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Survey Form</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+<div class="center">
+  <h1 id="title">Survey Form</h1>
+  <p id="description">
+    Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates inventore esse doloremque necessitati
+  </p>
```

Comment:
You use id twice in a div and I think it will have been better to your class instead of using id twice, you can just give a class something like <p class="description"> instead of <p id="description">

Reply:
- [ ] Please ensure to always state explicit reasons why you requested for a change; it might be obvious to you and not so for others. 😃 I think this statement is not complete, perhaps it's a mistake. Please kindly correct it. 👍